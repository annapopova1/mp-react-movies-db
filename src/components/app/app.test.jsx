import React from 'react';
import App from './app';

describe('<App/>', () => {
  test('should render App component', () => {
    const app = shallow(<App><div>test</div></App>);
    expect(app).toMatchSnapshot();
  });
});
