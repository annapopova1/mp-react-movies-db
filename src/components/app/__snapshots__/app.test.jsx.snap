// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should render App component 1`] = `
<ErrorHandler>
  <Header
    isMainPage={true}
  />
  <main
    className="container py-3"
  >
    <Connect(SearchPanelUI) />
    <NavPanel
      direction="right"
      links={
        Array [
          Object {
            "active": true,
            "handler": [Function],
            "param": "release_date",
            "title": "release date",
          },
          Object {
            "active": false,
            "handler": [Function],
            "param": "vote_average",
            "title": "rating",
          },
        ]
      }
      navBtn={null}
      primaryBrand="4 movies found"
      secondaryBrand="Sort by"
    />
    <MoviesList
      movies={
        Array [
          Movie {
            "budget": 0,
            "genres": Array [
              "genre1",
              "genre2",
            ],
            "id": 1,
            "overview": "This is a wider card with supporting text below as a natural lead-in to additional content.",
            "posterPath": "data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%22348%22%20height%3D%22225%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20348%20225%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_162cee39977%20text%20%7B%20fill%3A%23eceeef%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A17pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_162cee39977%22%3E%3Crect%20width%3D%22348%22%20height%3D%22225%22%20fill%3D%22%2355595c%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%22116.7265625%22%20y%3D%22120.3%22%3EThumbnail%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E",
            "releaseDate": "2010.01.10",
            "releaseYear": 2010,
            "revenue": 0,
            "runtime": 0,
            "tagline": "tagline",
            "title": "Test title",
            "voteAvg": 1.5,
            "voteCount": 0,
          },
          Movie {
            "budget": 0,
            "genres": Array [],
            "id": 2,
            "overview": "This is a wider card with supporting text below as a natural lead-in to additional content.",
            "posterPath": "data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%22348%22%20height%3D%22225%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20348%20225%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_162cee39977%20text%20%7B%20fill%3A%23eceeef%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A17pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_162cee39977%22%3E%3Crect%20width%3D%22348%22%20height%3D%22225%22%20fill%3D%22%2355595c%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%22116.7265625%22%20y%3D%22120.3%22%3EThumbnail%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E",
            "releaseDate": "2004.10.10",
            "releaseYear": 2004,
            "revenue": 0,
            "runtime": 0,
            "tagline": "tagline",
            "title": "Test title",
            "voteAvg": 4,
            "voteCount": 0,
          },
          Movie {
            "budget": 0,
            "genres": Array [
              "genre1",
              "genre2",
            ],
            "id": 3,
            "overview": "This is a wider card with supporting text below as a natural lead-in to additional content.",
            "posterPath": "data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%22348%22%20height%3D%22225%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20348%20225%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_162cee39977%20text%20%7B%20fill%3A%23eceeef%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A17pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_162cee39977%22%3E%3Crect%20width%3D%22348%22%20height%3D%22225%22%20fill%3D%22%2355595c%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%22116.7265625%22%20y%3D%22120.3%22%3EThumbnail%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E",
            "releaseDate": "2000.10.10",
            "releaseYear": 2000,
            "revenue": 0,
            "runtime": 0,
            "tagline": "tagline",
            "title": "Test title",
            "voteAvg": 1.1,
            "voteCount": 0,
          },
          Movie {
            "budget": 0,
            "genres": Array [
              "genre1",
            ],
            "id": 4,
            "overview": "This is a wider card with supporting text below as a natural lead-in to additional content.",
            "posterPath": "data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%22348%22%20height%3D%22225%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20348%20225%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_162cee39977%20text%20%7B%20fill%3A%23eceeef%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A17pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_162cee39977%22%3E%3Crect%20width%3D%22348%22%20height%3D%22225%22%20fill%3D%22%2355595c%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%22116.7265625%22%20y%3D%22120.3%22%3EThumbnail%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E",
            "releaseDate": "2018.10.10",
            "releaseYear": 2018,
            "revenue": 0,
            "runtime": 0,
            "tagline": "tagline",
            "title": "Test title",
            "voteAvg": 2,
            "voteCount": 0,
          },
        ]
      }
    />
  </main>
  <Footer />
</ErrorHandler>
`;
